staging/src/k8s.io/apiserver/pkg/apis/audit/types.go

```diff
@@ -33,14 +34,17 @@ const (
 	// LevelRequest provides Metadata level of auditing, and additionally
 	// logs the request object (does not apply for non-resource requests).
 	LevelRequest Level = "Request"
-	// LevelResponse provides Request level of auditing, and additionally
+	// LevelRequestResponse provides Request level of auditing, and additionally
 	// logs the response object (does not apply for non-resource requests).
-	LevelResponse Level = "Response"
+	LevelRequestResponse Level = "RequestResponse"
 )
 
 // Event captures all the information that can be included in an API audit log.
 type Event struct {
 	metav1.TypeMeta
+	// ObjectMeta is included for interoperability with API infrastructure.
+	// +optional
+	metav1.ObjectMeta
```

Comment:
> why? Which infrastructure?

Generation, generic clients, scope naming, API installer infrastructure (if we ever want it).  I've been down the path of excluding this before and you get whacky errors.  I don't see a reason to fight our infrastructure on a few fields, several of which we probably want in the end like annotations.